src/pages/signing_official_console/SigningOfficialResearchers2.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {useState, useEffect} from 'react';
+import {Notifications} from '../../libs/utils';
+import {Styles} from '../../libs/theme';
+import {User} from '../../libs/ajax/User';
+import {DAA} from '../../libs/ajax/DAA';
+import {DAC} from '../../libs/ajax/DAC';
+import { USER_ROLES } from '../../libs/utils';
+import SigningOfficialTable2 from './SigningOfficialTable2';
+
+export default function SigningOfficialResearchers2() {
```

Comment:
nit, naming. Can we name this something less derivative of what it will replace? It's already in a signing official specific directory, maybe just `ManageResearcherDAAs` or something similar. The text on the page doesn't mention Library Cards at all any more so this sort of makes sense.